amy/workshops/lookups.py

```diff
@@ -222,6 +222,31 @@ def get_queryset(self):
         return results
 
 
+
+class KnowledgeDomainLookupView(LoginNotRequiredMixin, AutoResponseView):
```

Comment:
Since this lookup is used in admin-restricted area of AMY, I'd like you to switch to admin required mixin. This will affect tests that you had to adjust for `LoginNotRequiredMixin`.